charts/opentelemetry-collector/templates/pdb.yaml

```
@@ -0,0 +1,14 @@
+{{- if .Values.podDisruptionBudget.enabled and .Values.standaloneCollector.enabled }}
```

This doesn't work :) Should be:
```suggestion
{{- if and .Values.podDisruptionBudget.enabled .Values.standaloneCollector.enabled }}
```

---

Thanks! Appreciate your feedback :)